doc/sphinx-guides/source/qa/overview.md

```diff
@@ -0,0 +1,30 @@
+# Overview
+
+```{contents} Contents:
+:local: 
+:depth: 3
+```
+
+## Introduction
+
+This guide describes the testing process used by QA at IQSS and provides a reference for others filling in for that role. Please note that many variations are possible, and the main thing is to catch bugs and provide a good quality product to the user community.
+
+## Workflow
+
+The basic workflow is as follows. Bugs or feature requests are submitted to GitHub by the community or by team members as issues. These issues are prioritized and added to a two-week sprint that is reflected on the GitHub {ref}`kanban-board`. As developers work on these issues, a GitHub branch is produced, code is contributed, and a pull request is made to merge these new changes back into the common {ref}`develop branch <develop-branch>` and ultimately released as part of the product. Before a pull request is moved to QA, it must be reviewed by a member of the development team from a coding perspective, and it must pass automated tests. There it is tested manually, exercising the UI (using three common browsers) and any business logic it implements.  Depending on whether the code modifies existing code or is completely new, a smoke test of core functionality is performed and some basic regression testing of modified or related code is performed. Any documentation provided is used to understand the feature and any assertions made in that documentation are tested. Once this passes and any bugs that are found are corrected, and the automated tests are confirmed to be passing, the PR is merged into the develop, the PR is closed, and the branch is deleted (if it is local). At this point, the PR moves from the QA column automatically into the Done column and the process repeats with the next PR until it is decided to {doc}`make a release </developers/making-releases>`.
```

Comment:
Where should we write about things to check AFTER a pull request is merged, such as:

- the Jenkins job that runs after the merge; did it pass? https://jenkins.dataverse.org/job/IQSS-dataverse-develop/
- the deployment to the beta server
- etc.